ruby/write_file.rb

```diff
@@ -0,0 +1,11 @@
+"""Script to write text to a new file.
+Run script as `ruby write_file.rb <output_file>.txt 'some text'`
+"""
+outfile = ARGV[0]
+text = ARGV[1]
+
+begin
+    File.open(outfile, 'w') do |f|
+        f.write(text.upcase)
```

Comment:
There's actually a method `File::write` that is more concise for doing this
(the `::` means it's a class method, as opposed to `File#write` which is an instance method) 

Here are the [docs for `IO::write`](https://docs.ruby-lang.org/en/master/IO.html#method-c-write) (which `File` inherits from `IO`)

Reply:
Nice. Will use this instead